wpimath/src/main/native/include/frc/controller/ProfiledPIDController.h

```diff
@@ -174,6 +174,16 @@ class ProfiledPIDController
     return m_controller.GetVelocityTolerance();
   }
 
+  /**
+   * Gets the accumulated error used in the integral calculation of this
```

Comment:
```suggestion
   * Gets the accumulated error used in the integral calculation of the
```
Kinda nitpicky but seems better

Reply:
"This" is usually better, and is used elsewhere in this class